static/scss/partials/modals.scss

```diff
@@ -164,6 +188,27 @@
     }
 }
 
+.t-domain-registration-success {
+
+    .c-modal-header {
+        padding-bottom: $layout-md;
+    }
+    
+    .t-domain-registration-body {
+
+        img {
+            position: relative;
+            margin-top: -$layout-md;
+            margin-bottom: $spacing-md;
```

Comment:
We can change this to `$layout-2xs` since its the same value as `$spacing-md` (to keep it consistent with the property above it)

Reply:
Oh! actually…`$layout-md` is the same as `$spacing-2xl`. Let's keep it spacing. Adjusted in 8d62d4797b20f3c3d12d995576a0f7728f227368

Reply:
<img width="618" alt="image" src="https://user-images.githubusercontent.com/2692333/140579340-375b15e4-2e73-401c-957c-c3af3d24dbd6.png">